amazon-security-lake/package.yaml

```diff
@@ -105,10 +105,32 @@ pipelines:
       subscribe "{{ inputs.topic }}"
       where @name == "ocsf.dns_activity"
       ocsf::apply
-      
+
       let $source_url = "{{ inputs.dns-activity-source-url }}"
 
       to_amazon_security_lake $source_url,
         region="{{ inputs.region }}",
         account_id="{{ inputs.account-id }}",
         timeout={{ inputs.timeout }}
+
+examples:
+  - name: Export with cross-account role
```

Comment:
Does it make sense to add another example here that does not make use of `external_id` (and maybe `role`)?

Reply:
That'd be nice!

Reply:
Yes, but it's non-trivial because we'd have to write additional documentation in the description how to set up the IAM roles in order to be able to use the example pipeline.

So let's wait until someone finds the time to add it in a future PR.